lib/zendesk_apps_support/validations/manifest.rb

```diff
@@ -65,6 +70,10 @@ def check_errors(error_types, collector, *checked_objects)
           end
         end
 
+        def ban_parameters(manifest)
```

Comment:
actually, there's other stuff we could assert missing / assert false from marketing apps if we wanted. Almost all the manifest fields in https://github.com/zendesk/zendesk_apps_support/blob/master/lib/zendesk_apps_support/manifest.rb#L7-L22 are irrelevant to marketing apps.